### DISCUSSION TITLE
What's the meaning of Editor SDKs configure?

### DISCUSSION THREAD

**Discussion by wenjoy** [yarnpkg/berry#3261] [Q&A]
Under the GETTING STARTED tab, there is a big topic about [Editor SDKS](https://yarnpkg.com/getting-started/editor-sdks).

I read the doc but I can't figure out :
1. What has it brought for me after I execute `yarn dlx @yarnpkg/sdks vscode`?
2. Why put this topic to a such outstanding position since yarn is a package manager tool? Why a package manager tool would care about user's editor tool? Isn't it weird?

**Comment by guotingchao** [comment]
Probably a hack to get the editor to adapt to the current zero-config。
I'm wondering the same thing.
